packages/api/README.md

```diff
@@ -55,11 +55,58 @@ You only need to `npm start` for subsequent runs. PR your env config to the wran
 
 ## API
 
-### 🔐 `PUT /car/:cid`
+The given API has a set of three different authentication levels:
 
-Upload a CAR for a root CID. _Authenticated_
+- 🤲 Public
+- 🔒 API or Magic Token
+- 👮 Magic Token (admin operations)
 
-### `GET /car/:cid`
+The 👮 API methods are only allowed with a Magic Token, and consequently only available via https://web3.storage
+
+### 🔒 `POST /car`
+
+Upload a CAR file for a root CID. _Authenticated_
+
+```console
+curl -X POST --data-binary @x.car -H 'Authorization: Bearer YOUR_API_KEY' http://127.0.0.1:8787/car -s | jq
```

Comment:
wondering if we should use the public API in these examples? I kept consistent with what was already here